app/src/Routes.tsx

```diff
@@ -55,7 +56,14 @@ const router = createBrowserRouter(
         </Route>
       </Route>
       <Route path="/projects" handle={{ crumb: () => "projects" }}>
-        <Route index element={<ProjectsPage />} />
+        <Route
+          index
+          element={
+            <ProjectStateProvider>
```

Comment:
If this state is not re-usable, it probably makes more sense to co-locate it with the page. In general though I think we can remove